dist/profile/templates/buildmaster/casc/clouds.yaml.erb

```diff
@@ -133,7 +133,7 @@ jenkins:
   - kubernetes:
       name: "<%= k8s_name %>"
       containerCap: <%= k8s_setup["max_capacity"] %>
-      credentialsId: "cik8s-kubeconfig"
+      credentialsId: "<%= k8s_setup["credentialsId"] %>"
```

Comment:
questionable if this needs to be configurable

Reply:
Same (the credential is different between ci.j, trusted.ci and others (both its id and the value), hence the need to configure it.)